access/access.go

```diff
@@ -136,13 +136,14 @@ type clt struct {
 	cancel context.CancelFunc
 }
 
-func NewClient(ctx context.Context, plugin string, addr string, tc *tls.Config) (Client, error) {
+// NewClient creates a new Teleport GRPC API client and returns it.
+//
+// TODO: Nate've removed the 2 seconds back-off time,
```

Comment:
Uh oh, I'll need to delete those commends if we keep the default options via the plugins app.go.